source/default_mode/Host.h

```diff
@@ -461,8 +461,7 @@ class Host: public Organism {
    * where n is the number of existing phage.
    */
   bool SymAllowedIn(){
-    bool do_phage_exclusion = my_config->PHAGE_EXCLUDE();
-    if(!do_phage_exclusion){
+    if(!my_config->SYM_EXCLUDE()){

```

Comment:
Even though you could not change this (after the alias is in), I think it's good to keep this name change so that it isn't confusing in the code why we are talking about phage